app.py

```diff
@@ -146,17 +165,17 @@ def publish_response(context, req, resp):
             "workspace": req['params']['workspace'],
             "actions": req['params']['actions'],
             "response": resp,
+            "services": resp.get("services", None),
+            "updatetag": resp.get("updatetag", None),
         }
 
         sns_helper = SNSLibrary(context)
-
-        if 'resultTopic' in req['params']:
```

Comment:
this should be 3 conditions:
1. `if body.get('services', None)`, then post results to `req['params']['requestTopic']` topic
2. `if 'resultTopic' in req['params']`, then post results to `req['params']['resultTopic']` topic
3. `else`, post results to `context.aws_inventory_sync_response_topic` topic.

Can you please make the adjustments?

Reply:
Please  review latest commit i have make changes in that also added the pagination for IAM